low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -223,8 +223,8 @@ def validate(self):
         pass
 
     def process(self):
-        # TODO: Import relevant python script here for further execution.
-        pass
+        from cortx.sspl.lowlevel.files.opt.seagate.sspl.setup import sspl_test
```

Comment:
Following is more readable
>from cortx.sspl.lowlevel.files.opt.seagate.sspl.setup.sspl_test import SSPLTestCmd
SSPLTestCmd(self.args).process()

Reply:
Done.